airflow/providers/google/cloud/hooks/bigquery.py

```diff
@@ -227,45 +236,43 @@ def create_empty_table(self,  # pylint: disable=too-many-arguments
         :type num_retries: int
         :return: None
         """
-        service = self.get_service()
-
-        project_id = project_id if project_id is not None else self.project_id
+        if num_retries:
+            warnings.warn("Parameter `num_retries` is deprecated", DeprecationWarning)
 
-        table_resource = {
+        _table_resource: Dict[str, Any] = {
```

Comment:
(With an eye to minimizing diff to make this easier to review), is there a reason to rename this variable?

Reply:
The reason is this line:
```
 table_resource = table_resource or _table_resource
```

I wanted to avoid adding a big if clause and building this simple dictionray everytime is not a big performance issue I think.